pkg/scm/git/git.go

```diff
@@ -34,7 +34,7 @@ type stiGit struct {
 	runner util.CommandRunner
 }
 
-var gitSshURLExp = regexp.MustCompile(`\A([\w\d\-_\.+]+@[\w\d\-_\.+]+:[\w\d\-_\.+%/]+\.git)$`)
+var gitSshURLExp = regexp.MustCompile(`\A([\w\d\-_\.+]+@[\w\d\-_\.+]+:[\w\d\-_\.+%/])`)
```

Comment:
Also, while you investigate, don't forget about IPv6:
http://www.chtaube.eu/news/2014/0918-git210-parser-bug-in-ipv6-ssh-urls/